cvprac/cvp_api.py

```diff
@@ -752,9 +756,10 @@ def apply_image_to_container(self, image, container):
                     Ex: {u'data': {u'status': u'success', u'taskIds': [u'32']}}
         '''
         return self.apply_image_to_element(image, container, container['name'],
-                                           'container')
+                                           'container', create_task)
 
-    def apply_image_to_element(self, image, element, name, id_type):
+    def apply_image_to_element(self, image, element, name, id_type,
+                               create_task):
```

Comment:
this is probably being a bit picky but should you set the default value of create_task = True here just in case this method is called directly so it will match the other edits?